src/Sentry/Instrumenter.cs

```diff
@@ -0,0 +1,17 @@
+namespace Sentry;
+
+/// <summary>
+/// Describes which approach is used to create spans.
+/// </summary>
+public enum Instrumenter
```

Comment:
I saw Java called it like that. It's a bit unfortunate because this is a rather generic name. Instrumenter of what? Tracing, error? Metrics?
No action here, just wanted to rant :)